kahypar/datastructure/hypergraph.h

```diff
@@ -1443,6 +1443,16 @@ class GenericHypergraph {
     }
   }
 
+  // ! Removes all fixed vertices
+  void resetFixedVertices() {
```

Comment:
Can you add a test case for this method in `kahypar/tests/datastructure/hypergraph_test.cc`?

Reply:
Done